core/src/main/java/com/linecorp/armeria/common/logging/DefaultRequestLog.java

```diff
@@ -166,7 +166,10 @@ private void propagateRequestSideLog(RequestLog child) {
         child.addListener(log -> requestHeaders(log.requestHeaders()), REQUEST_HEADERS);
         child.addListener(log -> requestContent(log.requestContent(), log.rawRequestContent()),
                           REQUEST_CONTENT);
-        child.addListener(log -> endRequest0(log.requestCause(), log.requestEndTimeNanos()), REQUEST_END);
+        child.addListener(log -> {
+            increaseRequestLength(log.requestLength());
```

Comment:
@tmdgus0118 You may want to rebase and update this part at #1574.

Reply:
How about using `requestLength(log.requestLength())` instead for readability?

Reply:
How about using `requestLength(log.requestLength())` instead for readability? Could it be different?

Reply:
No. Just forgot that we had it. Thanks :-)